main/apps/tvon/tvout-on.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/busybox sh
+
+touch /mnt/tvout
+killall -9 main
+sync
+
```

Comment:
doesn't seem to work without ``reboot``

Reply:
also how do you plan to switch between NTSC/PAL ?

Reply:
works without reboot with gmenux from source, or from you. Gmenux from 1.3.3 crash after restart

Reply:
oh, so we need to upload latest gmenu2x here (no need to use legacy 1.3.3, considering we are not even sure the src).